sycl/plugins/unified_runtime/ur/adapters/level_zero/adapter.cpp

```diff
@@ -19,6 +45,9 @@ urInit(ur_device_init_flags_t
        ur_loader_config_handle_t) {
   std::ignore = DeviceFlags;
 
+  // read variables from environment once
+  UrL0Debug = UrL0DebugReadEnvVar;
+
```

Comment:
Why no t just rename "UrL0DebugReadEnvVar" to "UrL0Debug"?

Reply:
UrL0DebugReadEnvVar is the lambda used to read the var from the environment. Value is read once here, and cached on UrL0Debug. So afterwards, adapter only needs to read the cached value from UrL0Debug, w/o needing to go again and read the environment using UrL0DebugReadEnvVar.

Reply:
you can still initialize a global (not static) UrL0Debug with a lambda that is only executed once, no?

Reply:
right, changing it to static.